extensions/Worker.Extensions.EventGrid/src/Properties/AssemblyInfo.cs

```diff
@@ -1,6 +1,6 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-﻿using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
 
-[assembly: ExtensionInformation("Microsoft.Azure.WebJobs.Extensions.EventGrid", "3.2.0")]
+[assembly: ExtensionInformation("Microsoft.Azure.WebJobs.Extensions.EventGrid", "3.2.1")]
```

Comment:
Don't we need to update the .csproj as well? 

https://github.com/Azure/azure-functions-dotnet-worker/blob/main/extensions/Worker.Extensions.EventGrid/src/Worker.Extensions.EventGrid.csproj#L9

Reply:
Yes, that change wasn't added to my commit. Fixed.